multiversx_sdk/core/transaction_computer.py

```diff
@@ -34,10 +34,10 @@ def compute_transaction_fee(self, transaction: ITransaction, network_config: INe
 
         return int(fee_for_move + processing_fee)
 
-    def compute_bytes_for_signing(self, transaction: ITransaction) -> bytes:
+    def compute_bytes_for_signing(self, transaction: ITransaction, with_signature: bool = False) -> bytes:
         self._ensure_fields(transaction)
 
-        dictionary = self._to_dictionary(transaction)
+        dictionary = self._to_dictionary(transaction, with_signature)
```

Comment:
and here it should directly call `self._to_dictionary(transaction, False)`

Reply:
since `False` is the default choice, can I leave it `dictionary = self._to_dictionary(transaction)`